### PR TITLE
Audit logging samples

### PR DESCRIPTION
**Pull request by greg-szrama**
Resolves https://github.com/redpanda-data/documentation-private/issues/2124

Added audit log message samples.

Preview: [New page](https://deploy-preview-141--redpanda-docs-preview.netlify.app/23.3/manage/audit-logging/audit-log-samples/)

### CODE REVIEW

File: modules/manage/pages/audit-logging/audit-log-samples.adoc [hunk 1]
```diff
@@ -0,0 +1,514 @@
...
+----
+{
+  "category_uid": 3,
+  "class_uid": 3002,
+  "metadata": {
+    "product": {
+      "name": "Redpanda",
+      "vendor_name": "Redpanda Data, Inc.",
+      "version": "v23.3.0-dev-2457-g76dc896f8c"
+    },
```
Comment (michael-redpanda) [inline modules/manage/pages/audit-logging/audit-log-samples.adoc]: FYI: New field added from when I originally gave you the schema

```suggestion
    "product": {
      "name": "Redpanda",
      // This is the Node ID of the broker that produced this audit event
      "uid": "2",
      "vendor_name": "Redpanda Data, Inc.",
      "version": "v23.3.0-dev-2457-g76dc896f8c"
    },
```

File: modules/manage/pages/audit-logging/audit-log-samples.adoc [hunk 2]
```diff
@@ -0,0 +1,514 @@
...
+  "type_uid": 300201,
+  "activity_id": 1,
+  "auth_protocol": "SASL-SCRAM",
+  "auth_protocol_id": 99,
+  "dst_endpoint": {
+    "ip": "127.0.0.1",
+    "port": 19092,
+    // Name of the Redpanda kafka server
+    "svc_name": "kafka rpc protocol"
+  },
```
Comment (michael-redpanda) [inline modules/manage/pages/audit-logging/audit-log-samples.adoc:51]:
```suggestion
  // This is the IP address of the Kafka broker that received the authorization request
  "dst_endpoint": {
    "ip": "127.0.0.1",
    "port": 19092,
    // Name of the Redpanda kafka server
    "svc_name": "kafka rpc protocol"
  },
```

File: modules/manage/pages/audit-logging/audit-log-samples.adoc [hunk 3]
```diff
@@ -0,0 +1,514 @@
...
+  "is_mfa": false,
+  "service": {
+    "name": "kafka rpc protocol"
+  },
+  "src_endpoint": {
+    "ip": "127.0.0.1",
+    // This is the client ID of the kafka client
+    "name": "rpk",
+    "port": 42906
+  },
```
Comment (michael-redpanda) [inline modules/manage/pages/audit-logging/audit-log-samples.adoc:64]:
```suggestion
  // This is the IP address of the client that generated the authorization request
  "src_endpoint": {
    "ip": "127.0.0.1",
    // This is the client ID of the kafka client
    "name": "rpk",
    "port": 42906
  },
```

File: modules/manage/pages/audit-logging/audit-log-samples.adoc [hunk 4]
```diff
@@ -0,0 +1,514 @@
+= Sample Audit Log Messages
+:description: Sample Redpanda audit log messages.
+
+Redpanda's audit logs comply with the Open Cybersecurity Schema Framework (OCSF). This provides a predictable and extensible solution that works seamlessly with industry standard tools. This page aggregates several sample log files covering a range of scenarios.
```
Comment (michael-redpanda) [inline modules/manage/pages/audit-logging/audit-log-samples.adoc]:
```suggestion
Redpanda's audit logs comply with version 1.0.0 of the Open Cybersecurity Schema Framework (OCSF) schema. This provides a predictable and extensible solution that works seamlessly with industry standard tools. This page aggregates several sample log files covering a range of scenarios.
```

File: modules/manage/pages/audit-logging/audit-log-samples.adoc [hunk 5]
```diff
@@ -0,0 +1,514 @@
...
+----
+{
+  "category_uid": 3,
+  "class_uid": 3002,
+  "metadata": {
+    "product": {
+      "name": "Redpanda",
+      "vendor_name": "Redpanda Data, Inc.",
+      "version": "v23.3.0-dev-2457-g76dc896f8c"
+    },
```
Comment (michael-redpanda) [inline modules/manage/pages/audit-logging/audit-log-samples.adoc]:
```suggestion
    "product": {
      "name": "Redpanda",
      "uid": "1",
      "vendor_name": "Redpanda Data, Inc.",
      "version": "v23.3.0-dev-2457-g76dc896f8c"
    },
```

File: modules/manage/pages/audit-logging/audit-log-samples.adoc [hunk 6]
```diff
@@ -0,0 +1,514 @@
...
+----
+{
+  "category_uid": 6,
+  "class_uid": 6003,
+  "metadata": {
+    "product": {
+      "name": "Redpanda",
+      "vendor_name": "Redpanda Data, Inc.",
+      "version": "v23.3.0-dev-2457-g76dc896f8c"
+    },
```
Comment (michael-redpanda) [inline modules/manage/pages/audit-logging/audit-log-samples.adoc:141]:
```suggestion
    "product": {
      "name": "Redpanda",
      "uid": "0",
      "vendor_name": "Redpanda Data, Inc.",
      "version": "v23.3.0-dev-2457-g76dc896f8c"
    },
```

File: modules/manage/pages/audit-logging/audit-log-samples.adoc [hunk 7]
```diff
@@ -0,0 +1,514 @@
...
+  "activity_id": 3,
+  "actor": {
+    "authorizations": [
+      {
+        "decision": "authorized",
+        // Represents a policy for a non-super user
+        "policy": {
+          "desc": "acl: {principal {type user name user} host {{any_host}} op all perm allow}, resource: type {topic}          name {*} pat
+tern {literal}",
+          "name": "aclAuthorization"
```
Comment (michael-redpanda) [inline modules/manage/pages/audit-logging/audit-log-samples.adoc]:
```suggestion
        "policy": {
          "desc": "acl: {principal {type user name user} host {{any_host}} op all perm allow}, resource: type {topic} name {*} pattern {literal}",
          "name": "aclAuthorization"
```

File: modules/manage/pages/audit-logging/audit-log-samples.adoc [hunk 8]
```diff
@@ -0,0 +1,514 @@
...
+  // within the period of the audit log collecting entries
+  // This provides start and end time (the time period these events were
+  // observed)
+  "count": 2,
+  "end_time": 1700533480725,
+  "metadata": {
+    "product": {
+      "name": "Redpanda",
+      "vendor_name": "Redpanda Data, Inc.",
+      "version": "v23.3.0-dev-2457-g76dc896f8c"
```
Comment (michael-redpanda) [inline modules/manage/pages/audit-logging/audit-log-samples.adoc]:
```suggestion
    "product": {
      "name": "Redpanda",
      "uid": "0",
      "vendor_name": "Redpanda Data, Inc.",
      "version": "v23.3.0-dev-2457-g76dc896f8c"
```

File: modules/manage/pages/audit-logging/audit-log-samples.adoc [hunk 9]
```diff
@@ -0,0 +1,514 @@
...
+----
+{
+  "category_uid": 6,
+  "class_uid": 6003,
+  "metadata": {
+    "product": {
+      "name": "Redpanda",
+      "vendor_name": "Redpanda Data, Inc.",
+      "version": "v23.3.0-dev-2457-g76dc896f8c"
+    },
```
Comment (michael-redpanda) [inline modules/manage/pages/audit-logging/audit-log-samples.adoc]:
```suggestion
    "product": {
      "name": "Redpanda",
      "uid": "2",
      "vendor_name": "Redpanda Data, Inc.",
      "version": "v23.3.0-dev-2457-g76dc896f8c"
    },
```

File: modules/manage/pages/audit-logging/audit-log-samples.adoc [hunk 10]
```diff
@@ -0,0 +1,514 @@
...
```
Comment (michael-redpanda) [inline modules/manage/pages/audit-logging/audit-log-samples.adoc]:
```suggestion
    "product": {
      "name": "Redpanda",
      "uid": "0",
      "vendor_name": "Redpanda Data, Inc.",
      "version": "v23.3.0-dev-2457-g76dc896f8c"
    },
```

File: modules/manage/pages/audit-logging/audit-log-samples.adoc [hunk 11]
```diff
@@ -0,0 +1,514 @@
+= Sample Audit Log Messages
+:description: Sample Redpanda audit log messages.
+
+Redpanda's audit logs comply with the Open Cybersecurity Schema Framework (OCSF). This provides a predictable and extensible solution that works seamlessly with industry standard tools. This page aggregates several sample log files covering a range of scenarios.
+
```
Comment (michael-redpanda) [inline modules/manage/pages/audit-logging/audit-log-samples.adoc:5]:
```suggestion
Redpanda produces the following OCSF class messages:

* Authentication (3002) for all authentication events
* Application Lifecycle (6002) for when the audit system is enabled or disabled or when Redpanda starts or stops (if auditing is enabled when Redpanda starts or stops)
* API Activity (6003) for any access to the Kafka API, Admin API or Schema Registry

Please refer to the [OCSF Schema Definition](https://schema.ocsf.io/) for more information about what the fields mean for each event class.
```

**Review by michael-redpanda — COMMENTED** [review]
Looks great!  I added a new field since the last time I gave you these examples so made some updates.

**Comment by michael-redpanda** [comment]
I should generate some SR events as well, but it's not a blocker imo

**Comment by Feediver1** [comment]
@greg-szrama For left navtree entries, we use init cap.
Once in a file, all headings are sentence style caps. 
Circle back to make sure your headings reflect this style. Thanks!

**Comment by greg-szrama** [comment]
Incorporated feedback into the updated page, ready for review!

**Review by graphcareful — APPROVED** [review]

**Comment by Feediver1** [comment]
Not sure why this PR is pulling in data transforms, install beta, and other PRs. May want to check with @Deflaimun

File: modules/manage/pages/audit-logging.adoc [hunk 12]
```diff
@@ -0,0 +1,66 @@
+= Audit Logging
+:description: Learn how to use Redpanda's audit logging capabilities.
+
+Many scenarios for streaming data include the need for fine-grained auditing of user activity related to the system. This is especially true for regulated industries such as finance, healthcare, and the public sector. Complying with PCI DSS v4 standards, for example, requires verbose and detailed activity auditing, alerting, and analysis capabilities.
```
Comment (Feediver1) [inline modules/manage/pages/audit-logging.adoc]: Already reviewed this content in https://github.com/redpanda-data/docs/pull/151

File: modules/manage/pages/audit-logging/audit-log-samples.adoc [hunk 13]
```diff
@@ -0,0 +1,531 @@
+= Sample Audit Log Messages
```
Comment (Feediver1) [inline modules/manage/pages/audit-logging/audit-log-samples.adoc:1]: Please revise either the navtree entry or the heading such that they match. Right now, the navtree has "Audit Log Message Examples", which is not the same as this heading.

File: modules/manage/pages/audit-logging/audit-log-samples.adoc [hunk 14]
```diff
@@ -0,0 +1,531 @@
...
+
+Redpanda's audit logs comply with version 1.0.0 of the https://github.com/ocsf[Open Cybersecurity Schema Framework (OCSF)]. This provides a predictable and extensible solution that works seamlessly with industry standard tools. This page aggregates several sample log files covering a range of scenarios.
+
+== Standard OCSF messages
+
+Redpanda produces the following standard OCSF class messages:
+
+* Authentication (3002) for all authentication events
+* Application Lifecycle (6002) for when the audit system is enabled or disabled or when Redpanda starts or stops (if auditing is enabled when Redpanda starts or stops)
+* API Activity (6003) for any access to the Kafka API, Admin API or Schema Registry
```
Comment (Feediver1) [inline modules/manage/pages/audit-logging/audit-log-samples.adoc]:
```suggestion
* API Activity (6003) for any access to the Kafka API, Admin API, or Schema Registry
```

File: modules/manage/pages/audit-logging/audit-log-samples.adoc [hunk 15]
```diff
@@ -0,0 +1,531 @@
...
+
+Please refer to the [OCSF Schema Definition](https://schema.ocsf.io/) for more information about what the fields mean for each event class.
```
Comment (Feediver1) [inline modules/manage/pages/audit-logging/audit-log-samples.adoc]:
```suggestion
Refer to the https://schema.ocsf.io/[OCSF Schema Definition] for the field definitions for each event class.
```

File: modules/manage/pages/audit-logging/audit-log-samples.adoc [hunk 16]
```diff
@@ -0,0 +1,531 @@
...
+
+== Authentication events
+
+These messages illustrate various scenarios around successful and unsuccessful authentication events.
+
+.Authentication successful
+[%collapsible]
+====
+This shows a basic scenario where an admin using rpk successfully authenticates with Redpanda. This event is an authentication type event.
```
Comment (Feediver1) [inline modules/manage/pages/audit-logging/audit-log-samples.adoc]:
```suggestion
This scenario shows how an admin using rpk successfully authenticates with Redpanda. This is an authentication type event.
```

File: modules/manage/pages/audit-logging/audit-log-samples.adoc [hunk 17]
```diff
@@ -0,0 +1,531 @@
...
+    "type_id": 1
+  }
+}
+----
+====
+
+.Authentication failed
+[%collapsible]
+====
+This illustrates a common failure scenario where an rpk user entered the wrong credentials. This is an authentication type event.
```
Comment (Feediver1) [inline modules/manage/pages/audit-logging/audit-log-samples.adoc]:
```suggestion
This scenario illustrates a common failure where a user entered the wrong credentials. This is an authentication type event.
```

File: modules/manage/pages/audit-logging/audit-log-samples.adoc [hunk 18]
```diff
@@ -0,0 +1,531 @@
...
+    "name": "admin",
+    "type_id": 1
+  }
+}
+----
+====
+
+== Kafka API events
+
+The Redpanda Kafka API offers a wide array of options for interacting with your Redpanda clusters. Here we have some messages from a few common interactions with the API.
```
Comment (Feediver1) [inline modules/manage/pages/audit-logging/audit-log-samples.adoc]:
```suggestion
The Redpanda Kafka API offers a wide array of options for interacting with your Redpanda clusters. Following are examples of messages from common interactions with the API.
```

File: modules/manage/pages/audit-logging/audit-log-samples.adoc [hunk 19]
```diff
@@ -0,0 +1,531 @@
...
+
+.Create ACL Entry
+[%collapsible]
+====
+This illustrates an access control list update which also requires a superuser authentication. It lists the ACL edited and the updated permissions. This is a management type event.
```
Comment (Feediver1) [inline modules/manage/pages/audit-logging/audit-log-samples.adoc]:
```suggestion
This example illustrates an ACL update that also requires a superuser authentication. It lists the edited ACL and the updated permissions. This is a management type event.
```

File: modules/manage/pages/audit-logging/audit-log-samples.adoc [hunk 20]
```diff
@@ -0,0 +1,531 @@
...
+      }
+    }
+  }
+}
+----
+====
+
+== Admin API events
+
+These illustrate some examples of audit messages related to use of the Redpanda Admin API.
```
Comment (Feediver1) [inline modules/manage/pages/audit-logging/audit-log-samples.adoc]:
```suggestion
The following examples show audit messages related to use of the Redpanda Admin API.
```

File: modules/manage/pages/audit-logging/audit-log-samples.adoc [hunk 21]
```diff
@@ -0,0 +1,531 @@
...
+
+.Requesting cluster configurations as a superuser
+[%collapsible]
+====
+This illustrates the log message from using the Admin API to retrieve the cluster configurations in a zipped archive. Of special note is that a user must authenticate with the superuser role to perform this action.
```
Comment (Feediver1) [inline modules/manage/pages/audit-logging/audit-log-samples.adoc]:
```suggestion
This example shows the log message when you use the Admin API to retrieve the cluster configurations in a zipped archive. Note that a user must authenticate with the superuser role to perform this action.
```

File: modules/manage/pages/audit-logging/audit-log-samples.adoc [hunk 22]
```diff
@@ -0,0 +1,531 @@
...
+  "status_id": 1,
+  "unmapped": {}
+}
+----
+====
+
+.Unauthorized user requesting cluster configurations
+[%collapsible]
+====
+Similar to the earlier example, this illustrates a user requesting cluster configurations as a zip archive. Unlike the previous example, however, the user in this case is not authorized to retrieve this information.
```
Comment (Feediver1) [inline modules/manage/pages/audit-logging/audit-log-samples.adoc]:
```suggestion
Similar to the previous example, this example illustrates a user requesting cluster configurations as a zip archive. Unlike the previous example, however, the user in this case is not authorized to retrieve this information.
```

**Review by Feediver1 — CHANGES_REQUESTED** [review]
See suggested updats

**Review by Feediver1 — APPROVED** [review]
lgtm!

**Review by graphcareful — APPROVED** [review]

**Review by deniscoady — APPROVED** [review]

**Pull request merged**